src/typeahead/index.js

```diff
@@ -105,6 +102,10 @@ var Typeahead = React.createClass({
     return result;
   },
 
+  visibleOptions: function(){
```

Comment:
Should be `getVisibleOptions()` for consistency with other accessors ?

Also, whitespace between `()` and `{`